docs/rpc.rst

```diff
@@ -0,0 +1,188 @@
+RPC
+===
+
+Coming soon: RPC docs powered by `slate <https://github.com/lord/slate>`__. Until then, read on.
```

Comment:
@melekes what does it take to host the slate docs live?

Reply:
I hosted the slate docs on Heroku just to provide an example. But it could be any server.

Reply:
#629